File: src/test/samples/invalidSamples/invalidSample.c
//#region FirstRegion
int x = 42;
//#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

// #region Second Region
class MyClass {
  //  #region    InnerRegion   
  void myMethod() {}
  //          #endregion  ends InnerRegion    

  // #region
  void myMethod2() {}
  //#endregion
};
// #endregion

File: src/test/samples/invalidSamples/invalidSample.rs
fn main() {
    // #region FirstRegion
    let x = 42;
    //#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

    //    #region Second Region
    struct MyClass;

    impl MyClass {
        // #region InnerRegion
        fn my_method(&self) {}
        //   #endregion   ends InnerRegion

        // #region
        fn my_method2(&self) {}
        //#endregion
    }
    //  #endregion
}
